### DISCUSSION TITLE
Slow browser rendering

### DISCUSSION THREAD

**Discussion by tangobango5** [cirruslabs/tart#481] [Q&A]
Hi, I am new to Tart. I am testing tart to run some automated tests(using selenium and web browser) in macos VMs, but am experiencing drastic slowness in the VMs compared to the host. Specifically the rendering is very slow. I have tried on mac mini and Macbook pro. Both gives poor performance for browser tests.

Is there a way to boost these browser tests?

**Comment by fkorotkov** [comment]
Could you please provide an example? What's the version of macOS you use on the host and for your VM?

Could you please also experiment if changing display resolution via `tart set --display 1920x1080` helps.
